modules/service-accounts-using-credentials-inside-a-container.adoc

```diff
@@ -26,14 +26,14 @@ by the token:
 +
```

Comment:
@adambkaplan Although this file is in the repo, it is currently commented-out, so that it's contents are not visible. Therefore, there is no preview of this content.

We should update the URL anyway, in case someone removes the comment and adds the content back to our documentation.